### PR TITLE
Feature bl 4614 signed up complete email

### PR DESCRIPTION
**Pull request by aaronlong**

### CODE REVIEW

File: webapp/src/main/java/uk/gov/dvsa/motr/notifications/service/NotifyService.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    public SendEmailResponse sendConfirmationEmail(
+            String emailAddress,
+            String registrationNumber,
+            LocalDate motExpiryDate,
+            String unsubscribeLink
+    ) throws NotificationClientException {
+
+        String templateId = config.getValue(CONFIRMATION_TEMPLATE_ID);
```
Comment (przemos) [inline webapp/src/main/java/uk/gov/dvsa/motr/notifications/service/NotifyService.java]: https://github.com/google/guice/wiki/InjectOnlyDirectDependencies

Comment (aaronlong) [inline webapp/src/main/java/uk/gov/dvsa/motr/notifications/service/NotifyService.java]: Fixed

File: webapp/src/main/java/uk/gov/dvsa/motr/web/eventlog/notify/NotifyEvent.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package uk.gov.dvsa.motr.web.eventlog.notify;
+
+import uk.gov.dvsa.motr.eventlog.Event;
+
+public abstract class NotifyEvent extends Event {
+
+    public NotifyEvent setEmail(String email) {
+
+        params.put("user-email", email);
```
Comment (przemos) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/eventlog/notify/NotifyEvent.java]: user-email/user-vrm ... maybe just email/vrm ?

Comment (aaronlong) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/eventlog/notify/NotifyEvent.java]: Sounds better :) fixed

File: webapp/build.gradle [hunk 3]
```diff
@@ -57,6 +58,7 @@ dependencies {
     compile 'javax.mail:mail:1.4.7'
     compile 'org.glassfish.jersey.media:jersey-media-json-jackson:2.25'
     compile 'org.glassfish.jersey.connectors:jersey-apache-connector:2.20'
+    compile('uk.gov.service.notify:notifications-java-client:3.0.0-RELEASE')
```
Comment (przemos) [inline webapp/build.gradle]: lets keep it without parenthesis like above

File: webapp/src/main/java/uk/gov/dvsa/motr/notifications/service/NotifyService.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+
+public class NotifyService {
+
+    private NotificationClient notificationClient;
+    private String templateId;
+
+    public NotifyService(NotificationClient notificationClient, String templateId) {
```
Comment (przemos) [inline webapp/src/main/java/uk/gov/dvsa/motr/notifications/service/NotifyService.java]: please make the name more specific: templateId - what is it used for ? confirmationTemplateId ?

Comment (aaronlong) [inline webapp/src/main/java/uk/gov/dvsa/motr/notifications/service/NotifyService.java]: fixed

File: webapp/build.gradle (hunk 3 above)
Comment (aaronlong) [inline webapp/build.gradle]: fixed

**Pull request closed without merging**